### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by v-zhuravlev**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -15,4 +17,3 @@
 /grafana-cloud-integration-utils/ @grafana/cloud-integrations
 /logs-lib/ @grafana/cloud-integrations
 /status-panels-lib/ @grafana/cloud-integrations
-/windows-observe-lib/ @grafana/cloud-integrations
```
Comment (Dasomeone) [inline CODEOWNERS:18]: Missing newline

Comment (v-zhuravlev) [reply CODEOWNERS:18]: it was added actually

File: CODEOWNERS [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 # Wildcard permissions should be loaded first
 /*-mixin/ @grafana/cloud-integrations
+/*-observ-lib/ @grafana/cloud-integrations
+/*-mixin/ @grafana/cloud-integrations
```
[the remark this answers is not in the document]
Comment (v-zhuravlev) [reply CODEOWNERS:7]: oops

**Review by Dasomeone — APPROVED** [review]
Hmm not sure why it's showing the newline issue in CODEOWNERS, but let's not block on it

**Pull request merged**